src/playground/Widgets/EmptyState/index.stories.tsx

```diff
@@ -10,16 +10,30 @@ const meta: Meta = {
   args: {
     title: "Performance",
     content: "See how Hugo's performance evolved over time",
-    buttons: [
-      { label: "Try it for free", variant: "promote" },
-      { label: "Book a demo", variant: "outline" },
-    ],
-    variant: "performance",
-    link: "/performance",
+    buttonLabel: "Try it for free",
+    buttonVariant: "promote",
+    icon: "area-graph",
   },
+  decorators: [
+    (Story) => (
+      <div className="w-[386px]">
```

Comment:
why do we need a fix width here? shouldn't the widget container provide sizing?

Reply:
The `WidgetContainer` goes full width. When the widget is placed in our layout with multiple columns it is auto-stretched to fit in it, but we're just showing this widget in the playground, that's why I provided this fixed width in the wrapper, so that it's closer to our design